examples/contact_form/src/main.rs

```diff
@@ -0,0 +1,39 @@
+#[macro_use] extern crate nickel;
+use nickel::{Nickel, HttpRouter};
+use std::collections::HashMap;
+
+// Required For Post
+extern crate hyper;
```

Comment:
I had received a weird issue without it the first time. But I cleaned and then recompiled and now no issue. I will remove it.